core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -387,139 +323,34 @@ private void applyCustomConfiguration(
         });
   }
 
-  protected void configureSecurityHandler(ServletContextHandler context) {
+  static void configureSecurityHandler(ServletContextHandler context, RestConfig config) {
```

Comment:
This is actually leftover from when ApplicationContext was an inner class to Application. In the end I decided to keep these static variants to offer some flexibility in code reuse. My thinking was that eventually we can decouple the server logic from the application logic entirely allowing Applications to use a generic of type `AbstractConfig` rather than the more restrictive RestConfig type. I don't have strong feelings either way though so we can revert it.